addons/account/models/sequence_mixin.py

```diff
@@ -247,6 +247,13 @@ def _set_next_sequence(self):
         self[self._sequence_field] = format.format(**format_values)
         self._compute_split_sequence()
 
+    def _is_first_from_seq_chain(self):
+        if not self.name or self.name == '/':
+            return False
+        _format, format_values = self._get_sequence_format_param(self.name)
+        return format_values['seq'] == 1
```

Comment:
`== 1` doesn't mean that it's the first of the chain, a sequence could start at 42

Reply:
True